pyproject.toml

```diff
@@ -128,6 +128,19 @@ exclude = [ # don't report on objects that match any of these regex
 ]
 override_SS05 = ['^Process ', '^Assess ', '^Access ']
 
+# --------------------
+# === Build config ===
+# --------------------
+[tool.cibuildwheel]
+skip = "pp* *_ppc64le *_i686 *_s390x cp313-*"
+build-frontend = "build[uv]"
+test-requires = ["pytest", "pytest-cov"]
+test-command = "pytest {project}/tests"
+[tool.cibuildwheel.macos]
+before-all = ["brew install uv libomp"]
+environment = { MACOSX_DEPLOYMENT_TARGET = "14.0" }
```

Comment:
You are hardcoding a minimum macOS target of macOS 14 but trying to build on macOS 13. I'd recommend something like 11 or 10.13 here.

Also, don't install uv with brew, install it in the GitHub action with setup-uv.

Reply:
Thank you for your review!
Actually, 'libomp' makes it fix the version.
Do you know any workaround to install it?

Reply:
You cannot use brew packages when building libraries (unless they are CLI only, like cmake), brew packages are not designed to be redistributable. You have to build libomp yourself with an older macOS target. This is the most recent example I recall: https://github.com/thaler-lab/Wasserstein/blob/7f658fd6c62d13d9e2f40f6f015fd302bbfc9249/.github/workflows/build-wheels.yml#L46

Reply:
That would be very helpful!
I will try to revise my workflow.

Reply:
It's probably better to install it into a local directory and then add that directory to meson's search path. Not very familiar with how to do that in meson, though.